NSS-Day-Cohort-69/tabloid-one-more-div
# Feature - React to a Post

### Purpose
To allow Users to React and Unreact to a Post

### Files Changed
- Created file and wrote initial Post Reaction DTO class for submitting a new Post Reaction in new `PostReactionCreateDTO.cs` DTO class
- Created file, and added endpoints for creating and deleting Post Reactions in new `PostReactionController.cs` controller
- Created file, and wrote fetches for creating and deleting Post Reactions in new `postReactionManager.js` manager
- Added logic for Reacting and Unreacting to a Post in `PostDetails.jsx`

### To Test
Fetch, setup, and run according to project README, then confirm the following
- Clicking on a grey Reaction button in a Post Details view turns the button blue, the number increments, and a PostReaction is created for that User/Post/Reaction
- Clicking on a blue Reaction button in a Post Details view turns the button grey, the number decrements, and the related PostReaction is deleted

closes #13
